packages/ngtools/webpack/src/ivy/paths.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+import * as nodePath from 'path';
+
+const normalizationCache = new Map<string, string>();
+
+export function normalizePath(path: string): string {
+  let result = normalizationCache.get(path);
+
+  if (result === undefined) {
+    result = nodePath.win32.normalize(path).replace(/\\/g, nodePath.posix.sep);
```

Comment:
Why not just `path.normalize` for posix systems?

Reply:
There's no guarantee on the input paths.  They could be a mix of directory separators; potentially doubled up as well.  The Windows normalize variant will guarantee only the default Windows directory separator.  This in turn reduces the complexity of the replace and provides a stronger guarantee on the output.